examples/react-vanilla-ts-src/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "lib": ["es2022", "DOM"],
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "types": ["node"],
+    "strict": true,
+    "noUnusedLocals": true,
+    "declaration": true,
+    "isolatedDeclarations": false,
+    "esModuleInterop": true,
+    "verbatimModuleSyntax": true,
+    "skipLibCheck": true,
+    "outDir": "dist",
+    "jsx": "react-jsx",
+  },
+  "include": [
+    "./globals.d.ts", 
+    "./src/**/*.d.ts", 
+    "./src/**/*.ts",
+  ],
+  "exclude": ["**/*.test.ts"]
+}
```

Comment:
I generally like to have two `tsconfig.json` files: one for the server code and one for the client code. I can add another PR on top of this for that.